include/oneapi/dpl/internal/function.h

```diff
@@ -31,25 +31,22 @@ namespace internal
 {
 using ::std::get;
 
-#if _ONEDPL_BACKEND_SYCL
-using oneapi::dpl::__internal::is_hetero_iterator;
-using oneapi::dpl::__par_backend_hetero::__internal::__buffer;
-#endif
-
 #if _ONEDPL_BACKEND_SYCL
 // Helpers used to get indexable access to the data passed to the SYCL implementation of an
 // algorithm from either a SYCL iterator or a USM pointer.
 template <sycl::access::mode Mode, typename Policy, typename Iterator>
 auto
-get_access(Policy, Iterator i, ::std::enable_if_t<is_hetero_iterator<Iterator>::value>* = nullptr)
+get_access(Policy, Iterator i,
```

Comment:
About `get_access(Policy, Iterator i......` It seems unused code (39 and 48 lines). Lets remove it(?)

Reply:
Issue created: https://github.com/oneapi-src/oneDPL/issues/1199

Reply:
Fixed (removed)